src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ThreadingExtensions.cs

```diff
@@ -26,7 +26,7 @@ internal static class ThreadingExtensions
         /// <exception cref="OperationCanceledException">
         ///     The result is awaited and <paramref name="cancellationToken"/> is cancelled.
         /// </exception>
-        public static JoinableTaskFactory.MainThreadAwaitable SwitchToUIThread(this IProjectThreadingService threading, CancellationToken cancellationToken)
```

Comment:
This was deliberate so that it would overload existing method.

Reply:
Ok, I'll back this out and add a comment to that effect.

Reply:
#5790